services/web/server/src/simcore_service_webserver/exporter/file_downloader.py

```diff
@@ -38,19 +27,16 @@ async def append_file(self, link: str, download_path: Path):
 
     async def download_files(self, app: Application):
         """starts the download and waits for all files to finish"""
-
-        # run this async, parfive will support aiofiles in the future as stated above
-        wrapped_function = aiofiles_os.wrap(self.downloader.download)
         exporter_settings = get_settings(app)
-        results = await wrapped_function(
+        results = await self.downloader.run_download(
             timeouts={
                 "total": exporter_settings.downloader_max_timeout_seconds,
                 "sock_read": 90,  # default as in parfive code
             }
         )
-        log.debug("Download results %s", results)
 
-        if len(results) != self.total_files_added:
+        log.debug("Download results %s", results)
```

Comment:
MINOR: for debugging it is very handy to use the new f-string feature in 3.8 to include the name of the variable directly in the message. e,g,

```python
log.debug("Downloading %s using %s", f"{results=}", f"{self.downloader=})
```

provided that  ``Downloader`` implements ``__str__`` with some human-friendly message